scripts/backup.sh.in

```diff
@@ -178,10 +179,24 @@ replicate_archives() {
 	saltenv=metalk8s-@@VERSION
 }
 
+clean_old_archives() {
+    local -r archive_dir="$(dirname "$BACKUP_ARCHIVE")"
+
+    mapfile -t archives < <(ls -t "$archive_dir"/*.tar.gz)
+    local -r nb_archives="${#archives[@]}"
+
+    if (( nb_archives > NB_BACKUPS )); then
+        for (( i=NB_BACKUPS; i<nb_archives; i++ )); do
+            rm -f "$archive_dir/${archives[$i]}"
+        done
+    fi
+}
```

Comment:
It will not be enough, the backups are copied to remote nodes, so this cleanup also needs to be done on other nodes

Reply:
even if the cleanup is done before replicating the backups ?